admin/listPurchases/listView.html

```diff
@@ -267,6 +267,39 @@
     `;
 }
 
+function markOrderDispatched(id){    
+    let dispatchButton = document.querySelector(".more-info-" + id + " .mark-as-sent-btn");
+    dispatchButton.classList.add("isClicked");
+    requestMarkOrderDispatched(id, (error) => {
+        if(error) {
+            document.write(error.response);
```

Comment:
This isn't a great way to report errors... An `alert` would be better but still bad. We probably need some general error reporting modal, right? Would also be nice to reuse else where.

For now, replace with alert- let's do better error handling in a later PR